libs/common/src/constants/record-correction-status.ts

```diff
@@ -0,0 +1,13 @@
+export type RecordCorrectionDisplayStatus = 'Record correction sent' | 'Record correction received';
+
+export type RecordCorrectionStatus = 'RECORD_CORRECTION_SENT' | 'RECORD_CORRECTION_RECEIVED';
+
+export const RECORD_CORRECTION_DISPLAY_STATUS: { SENT: RecordCorrectionDisplayStatus; RECEIVED: RecordCorrectionDisplayStatus } = {
+  SENT: 'Record correction sent',
+  RECEIVED: 'Record correction received',
+};
+
+export const RECORD_CORRECTION_STATUS: { SENT: RecordCorrectionStatus; RECEIVED: RecordCorrectionStatus } = {
+  SENT: 'RECORD_CORRECTION_SENT',
+  RECEIVED: 'RECORD_CORRECTION_RECEIVED',
+};
```

Comment:
I think usually we define the const and then extract the type from it in another file, e.g:
```typescript
// in a file in the constants folder
export const RECORD_CORRECTION_STATUS = {
  SENT: 'RECORD_CORRECTION_SENT',
  RECEIVED: 'RECORD_CORRECTION_RECEIVED',
} as const;

// In a file in the types folder
export type RecordCorrectionStatus = ValuesOf<typeof RECORD_CORRECTION_STATUS>;
```
I think we should be consistent with existing patterns please

Reply:
done